resources/js/src/app/shared/interfaces.ts

```diff
@@ -32,9 +32,14 @@ export interface Facet {
 }
 
 export interface FacetValue {
+    count?: string;
+    image?: string;
```

Comment:
I think that `image` should only be available for `ColorFacetValue`?

Reply:
All the facet calues get this property, only that it's an empty string usually. Still, it exists on everything.

Reply:
Hm.. that's odd, that doesn't really make sense. Should this even be passed as an empty string to non-image filters anyway? Maybe we can refactor this a bit.

Reply:
Doesn't seem like it's being used anywhere either and the `ColorFacetValue` has a separate property that contains the same value. I'm removing it then.